docs/source/virtual_ecosystem/virtual_ecosystem_in_use.md

```diff
@@ -0,0 +1,299 @@
+to run a simulation. The following command line arguments set up the example data
+directory in Linux, Mac or Windows Subsystem for Linux (WSL).
+
+```{code-cell}
+import pathlib
+
+import matplotlib.pyplot as plt
+import matplotlib.dates as mdates
+import numpy as np
+import xarray
+```
+
+```{code-cell}
+:tags: [remove-cell]
+
+%%bash
+# Remove any existing VE data directory in the /tmp/ directory
+if [ -d /tmp/ve_example ]; then
+  rm -r /tmp/ve_example
+fi
+```
+
+```{code-cell}
+%%bash
+# Install the example data directory from the Virtual Ecosystem package
+ve_run --install-example /tmp/
+```
+
+The `ve_example` directory contains the following files. These include:
+
+* the `config` directory of TOML format configuration files,
+* the `data` and `source` directories of netCDF format data files,
+* the `generation_scripts` directory containing example recipes for generating files, and
+* the `out` directory, which will be used to store model outputs.
+
+```{code-cell}
+# Get a generator of files in the example directory
+example_files = (p for p in pathlib.Path("/tmp/ve_example/").rglob("*") if p.is_file())
+
+# Print the relative paths of files
+for file in example_files:
+    print(file.relative_to("/tmp/ve_example"))
+```
+
+## Run the Virtual Ecosystem model
+
+Now the example data and configuration have been set up, the `ve_run` command can be
+used to execute a Virtual Ecosystem simulation. The `progress` option shows the progress
+of the simulation through the various modelling stages.
+
+```{code-cell}
+%%bash
+ve_run /tmp/ve_example/config \
+    --out /tmp/ve_example/out \
+    --logfile /tmp/ve_example/out/logfile.log \
+    --progress \
+```
+
+The log file is very long and shows the process of running the model. The code below
+shows the start and end lines from the log to give and idea of what it contains.
+
+```{code-cell}
+# Open and read the log
+with open("/tmp/ve_example/out/logfile.log") as log:
+    log_entries = log.readlines()
+
+# Print the first lines
+for entry in log_entries[:6]:
+    print(entry.strip())
+
+print("...")
+
+# Print the last lines
+for entry in log_entries[-5:]:
+    print(entry.strip())
+```
+
+## Looking at the results
+
+The Virtual Ecosystem writes out a number of data files:
+
+* `initial_state.nc`: A single compiled file of the initial input data.
+* `all_continuous_data.nc`: An optional record of time series data of the variables
+  updated at each time step.
+* `final_state.nc`: The model data state at the end of the final step.
+
+These files are written to the standard NetCDF data file format.
+
+```{code-cell}
+# Load the generated data files
+initial_state = xarray.load_dataset("/tmp/ve_example/out/initial_state.nc")
+continuous_data = xarray.load_dataset("/tmp/ve_example/out/all_continuous_data.nc")
+final_state = xarray.load_dataset("/tmp/ve_example/out/final_state.nc")
+```
+
+### Initial state
+
+The `initial_state.nc` file contains all of the data required to start up the model.
```

Comment:
Add something like: It also contains time series of climate variables at reference height that drive the model.